### PR TITLE
Fix lightning NOx emission bug

### PR DESCRIPTION
**Pull request by tangq**
During v3atm aerosol-chemistry integration, "NO" was removed from the external foring list of chemistry mechanism files ending in _vbs.in. This means that the lightning NOx emissions won't be used by the simulations using these mechanisms. This PR corrects this bug.

[non-BFB]

-------------------------------------
The test simulation was discussed at the [v3atm integration meeting](https://acme-climate.atlassian.net/wiki/spaces/ATMOS/pages/3854434386/2023-07-12+Meeting+notes+-+v3+Integration) and documented at this confluence [page](https://acme-climate.atlassian.net/wiki/spaces/ATMOS/pages/3849715713/20230707.NO-lght-fix.5x.v3alpha02.F2010.chrysalis).

### CODE REVIEW

**Review by mingxuanwupnnl — APPROVED** [review]
Looks good to me. Thank you for fixing it.

**Comment by susburrows** [comment]
@tangq I labeled this as a "bugfix PR" based on your description.  Should it also be marked "CC"?

**Review by keziming — APPROVED** [review]

**Comment by susburrows** [comment]
@rljacob how is "climate-changing" defined for PRs?  Is it based on the MVK test?

**Comment by rljacob** [comment]
Usually we leave it to the feature developer.  The should know.  Passing the MVK test is enough to say it is NOT climate changing but we don't have a test that unambiguously says a PR IS climate changing.

**Comment by crterai** [comment]
I've looked at the side-by-side comparisons between IICE and I urge others to look at it as well. 
From what I see, there's a 0.19 Wm-2 decrease in FSNTOA, which is somewhat balanced by a 0.05 Wm-2 change in FLNT. So RESTOM changes are 0.14 Wm-2. Precipitation patterns are slightly changed, where differences are most evident in Western Pacific, but these are 11 year runs, and my sense is that this is something that can be tuned with, if it's deemed unacceptable. 
Winds, temperature, and RH profiles all look similar as well. 
All in all, I think the change is acceptable given that it's a bug fix. 

https://portal.nersc.gov/project/m2136/bin/iice/iice.cgi?url1=https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.golaz/E3SMv3_dev/20230620.v3alpha01-ncautoexp10.F2010.chrysalis/e3sm_diags/atm_monthly_180x360_aave/model_vs_obs_0002-0011/viewer&label1=Baseline&url2=https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.zke/E3SMv3_dev/20230707.NO-lght-fix.1x.v3alpha02.F2010.chrysalis/e3sm_diags/atm_monthly_180x360_aave/model_vs_obs_0002-0011/viewer&label2=with%20NO%20fix&url3=&label3=&url4=&label4=&category=&plot=&diff=0&dconfig=vsObs&

**Comment by rljacob** [comment]
Oh right.  Labeling doesn't matter for a feature branch PR like this.  The PR to master is what needs to be labeled correctly.

**Comment by rljacob** [comment]
I'm also removing "bug fix PR" label because its the PR to master that fixes the bug.  Is this bug even on master?

**Comment by susburrows** [comment]
My @mahf708 , I also would have classified this as "CC" based on the definition in the [E3SM Code Review and New Feature Request Process](https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/3438608385/E3SM+Code+Review+and+New+Feature+Process#Code-changes-are-divided-into-4-categories), which is:

"**Climate changing**

- New feature is brought into the model turned on

- Other model configuration change (parameters, resolution, forcing data)

github PR label:  CC"

This PR changes an input data set, so I would have thought it is considered CC by default.  @rljacob can you clarify if that is not the case?

It also isn't clear that to me this would converge to an identical climate as the prior commit.  I'll add some more comments on this point shortly based on my review of the IICE viewer.

**Comment by susburrows** [comment]
Could this be added to the big PR to master to simplify things @wlin7 ?

**Comment by susburrows** [comment]
I also agree with @mahf708 that just because a change in climate is acceptable and needed, that doesn't imply that the PR should not be labelled as CC.  The purpose of the CC label is so that others in the project (for example, colleagues in different component groups) can clearly understand when the answers should be expected to change.

**Comment by rljacob** [comment]
We already know that master_MAM5_wetaero_chemdyg will be CC.  Adding this just makes it more CC but all we care about in the end is how master_MAM5_wetaero_chemdyg is climate changing "in total".   YOU may need to keep it straight while preparing master_MAM5_wetaero_chemdyg for integration but the feature review process doesn't apply for a branch-to-branch PR.

**Comment by rljacob** [comment]
Yes you can label it CC if you want to help with the record keeping.

**Comment by susburrows** [comment]
Thanks for clarifying that, @rljacob

**Comment by susburrows** [comment]
@tangq , did you mean to link to [this Confluence page](https://acme-climate.atlassian.net/wiki/spaces/ATMOS/pages/3849715713/20230707.NO-lght-fix.5x.v3alpha02.F2010.chrysalis) in the PR description?  The link you provided points to a page describing the 5xlightning NOx production test, which I think is a different run.

Thanks for providing that link to the IICE viewer, Chris.  One thing I notice in the IICE viewer diagnostics you linked to is a [ mean change of -0.69 W/m2](https://portal.nersc.gov/project/m2136/bin/iice/iice.cgi?url1=https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.golaz/E3SMv3_dev/20230620.v3alpha01-ncautoexp10.F2010.chrysalis/e3sm_diags/atm_monthly_180x360_aave/model_vs_obs_0002-0011/viewer&label1=Baseline&url2=https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.zke/E3SMv3_dev/20230707.NO-lght-fix.1x.v3alpha02.F2010.chrysalis/e3sm_diags/atm_monthly_180x360_aave/model_vs_obs_0002-0011/viewer&label2=with%20NO%20fix&url3=&label3=&url4=&label4=&category=display&plot=polar/CERES-EBAF-surface-v4.1/ceres_ebaf_surface_v4.1-FLDSC-ANN-polar_S.png&diff=1&dconfig=vsObs&) in clear-sky downwelling LW over the SH polar region, and an [mean SH polar increase of 0.2 W/m2](https://portal.nersc.gov/project/m2136/bin/iice/iice.cgi?url1=https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.golaz/E3SMv3_dev/20230620.v3alpha01-ncautoexp10.F2010.chrysalis/e3sm_diags/atm_monthly_180x360_aave/model_vs_obs_0002-0011/viewer&label1=Baseline&url2=https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.zke/E3SMv3_dev/20230707.NO-lght-fix.1x.v3alpha02.F2010.chrysalis/e3sm_diags/atm_monthly_180x360_aave/model_vs_obs_0002-0011/viewer&label2=with%20NO%20fix&url3=&label3=&url4=&label4=&category=display&plot=polar/CERES-EBAF-surface-v4.1/ceres_ebaf_surface_v4.1-FSDSC-ANN-polar_S.png&diff=0&dconfig=vsObs&) in the downwelling clearsky solar flux.

This might be related to the [reduced spring/summer SCO](https://portal.nersc.gov/project/m2136/bin/iice/iice.cgi?url1=https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.golaz/E3SMv3_dev/20230620.v3alpha01-ncautoexp10.F2010.chrysalis/e3sm_diags/atm_monthly_180x360_aave/model_vs_obs_0002-0011/viewer&label1=Baseline&url2=https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.zke/E3SMv3_dev/20230707.NO-lght-fix.1x.v3alpha02.F2010.chrysalis/e3sm_diags/atm_monthly_180x360_aave/model_vs_obs_0002-0011/viewer&label2=with%20NO%20fix&url3=&label3=&url4=&label4=&category=display&plot=annual_cycle_zonal_mean/OMI-MLS/OMI-MLS-SCO-Annual-Cycle.png&diff=1&dconfig=vsObs&) over the South Pole in these runs.  (this would be more easily discernable on an mvm plot if it were available, but you can make out in the IICE viewer that there are some differences).

In the links provided on the Confluence page for this PR, I was not able to find other diagnostics that would enable me to understand how the stratospheric ozone has been impacted by this PR, except for the SCO annual cycle comparison linked above.  The stratospheric plot set can be used to produce mvm plots of the zonal mean ozone ([example](https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.ngmahfouz/E3SMv3_dev/pd.free.3b3a953.v3alpha02.plus0K.chrysalis/e3sm_diags/atm_monthly_180x360_aave_mvm/model_vs_model_2005-2014/viewer/zonal_mean_2d_stratosphere/index.html)), but it appears not to have been produced in the e3sm_diags output for this run. 

I know that e3sm_diags can be configured to produce these diagnostics, for example see the ERFtracker version for v3alpha02 [here](https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.ngmahfouz/E3SMv3_dev/pd.free.3b3a953.v3alpha02.plus0K.chrysalis/e3sm_diags/atm_monthly_180x360_aave_mvm/model_vs_model_2005-2014/viewer/).

Since it's clear that the stratospheric chemistry can also be sensitive to changes to the tropospheric chemistry, would it be possible to ensure that the stratospheric plot set is always turned on for all future tests of changes to the chemistry (including mvm mode)?

I also am commenting here more extensively because want to raise out collective awareness as a team that when changes are made to the chemistry (or impact the chemistry), there are other diagnostics we need to check routinely, beyond the typical climate variables such as the global mean RESTOM, PRECT.  For example, I would suggest that any time the chemistry is changed we need to be checking for the impacts of those changes on (1) aerosols, (2) tropospheric chemistry -- ideally extending beyond ozone to include major oxidant species like OH and NOx (3) stratospheric chemistry (including ozone and specific/relative humidity), (4) any significant changes to stratospheric chemistry or radiation budgets over the poles, (5) any concomitant changes in temperature (especially in the UT/LS region) and circulation (including changes to the polar jets or Hadley cells) that are induced by the changes in the stratospheric chemistry.  The same checks probably need to be done for any developments that are expected to have potentially significant impacts on chemistry, such as any development or tuning that results in changes to convection.

I am not sure whether these differences in downwelling LW are large enough to appreciably change the Antarctic surface mass balance in a coupled simulation, but we have seen examples in recent coupled runs where changes in the stratospheric chemistry had appreciable impacts on the cryosphere, so this is something to be cognizant of and keep an eye on.

**Comment by mingxuanwupnnl** [comment]
Just want to clarify a bit. I think there are two aspects discussed or involved here.

1. Add NO to chem mech files, so that NO lightning emission will be added. It is explicitly included in this PR. We can check the results on https://acme-climate.atlassian.net/wiki/spaces/ATMOS/pages/3849650366/20230707.NO-lght-fix.1x.v3alpha02.F2010.chrysalis

2. Tuning NO lightning emission up to reduce TCO bias. So changing lght_no_prd_factor = 1.0 to 5.0. It is not explicitly included in this PR, as the changing of the factor is in runscript for namelist. The results are shown on https://acme-climate.atlassian.net/wiki/spaces/ATMOS/pages/3849715713/20230707.NO-lght-fix.5x.v3alpha02.F2010.chrysalis

For the first part, the changes in E3SM_diags are small. It is a bug fix. We may want to include it in v3alpha02 branch.
For the second part, it is more related to tuning, and we may do more experiments to finalize this factor, although  lght_no_prd_factor = 5.0 looks good enough.

**Comment by susburrows** [comment]
Thanks for the clarification @mingxuanwupnnl - that is helpful!

**Comment by tangq** [comment]
> We already know that master_MAM5_wetaero_chemdyg will be CC. Adding this just makes it more CC but all we care about in the end is how master_MAM5_wetaero_chemdyg is climate changing "in total". YOU may need to keep it straight while preparing master_MAM5_wetaero_chemdyg for integration but the feature review process doesn't apply for a branch-to-branch PR.

@rljacob , thanks for the guidance , which the chemistry team will happily follow. The key point of this PR is to fix an inadvertent bug in master_MAM5_wetaero_chemdyg and better prepare it for integration.

**Comment by susburrows** [comment]
The findings @mahf708 reports above seem to me to be consistent with what we should expect from this bug-fix PR, and I concur with @crterai that the impacts are small enough that we don't need further evaluation, given that these impacts will be tested as part of the overall master_MAM5_wetaero_chemdyg branch before merging to master.

However, I would like to request that the "big PR" description clearly documents that this change is included, and links back to this PR (and maybe also the relevant Confluence page) so that it is clear to others that this change is included when the branch is merged to master.

**Comment by wlin7** [comment]
Merged to v3atm/eam/master_MAM5_wetaero_chemdyg.

This bug also exists in current master since the merge of #5625 (to couple VBS SOA with chemUCI), in the mechanism file [pp_chemUCI_linozv3_mam4_resus_mom_vbs.in](https://github.com/E3SM-Project/E3SM/pull/5625/files#diff-3a5aa86fadff71ca895055c955f0f9130bf5a1d4dcbec1d94d36f1b49d39638d), which is then carried to the [mechanism file](https://github.com/E3SM-Project/E3SM/blob/v3atm/eam/master_MAM5_wetaero_chemdyg/components/eam/chem_proc/inputs/pp_chemUCI_linozv3_mam5_vbs.in) used by the full v3atm config. 

Note that the mechanism file used by #4707 for standalone chemUCI-Linozv3 compset does not have this bug, neither the one used by chemUCI-Linozv3-mam5.

**Pull request merged**
